cmd/delete.go

```diff
@@ -101,8 +102,35 @@ func RunDelete(options *DeleteOptions) error {
 		return fmt.Errorf("Unable to init reconciler: %v", err)
 	}
 
-	if err := reconciler.Destroy(); err != nil {
-		return fmt.Errorf("Unable to destroy resources for cluster [%s]: %v", options.Name, err)
+	done := make(chan bool)
+	message := make(chan error)
+
+	go func() {
+		message <- reconciler.Destroy()
+	}()
+
+	go func(description string, symbol string, c chan bool) {
+		if description != "" {
+			fmt.Print(description)
```

Comment:
I've just seen that `logger.Level` is initialized to 2, so users wouldn't see the process activity indicator unless they move to at least warning level. `logger.Always` isn't the best option either because of the hard-wired ✿ label. What are your thoughts on introducing a function `logger.Log` that ignores `logger.Level` and that does not apply any label?